Makefile

```
@@ -659,6 +659,13 @@ install-git-hooks:
 	    echo "done."; \
 	fi
 
+# Rules for updating github workflows.
+workflows: .github/workflows/verify.yml
```

nit:
```suggestion
PHONY: workflows
workflows: .github/workflows/verify.yml
```
Another nit: I'd probably rename this to `update-workflows` or `generate-workflows`

Just thinking that the makefile starts to be quite convoluted. It might be good to add some high-level "generate" target to auto-(re-)generate everything in the future. But not in this PR....

---

Updated accordingly.

---

> Just thinking that the makefile starts to be quite convoluted. It might be good to add some high-level "generate" target to auto-(re-)generate everything in the future. But not in this PR....

Yeah, I noticed and was thinking about that, too. Then quickly decided that definitely not in this PR...